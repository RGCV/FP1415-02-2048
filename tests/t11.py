## ========================================================================== ##
## -------------------------------------------------------------------------- ##
##                                 TEST 11                                    ##
## -------------------------------------------------------------------------- ##
## ========================================================================== ##

# -- Best: request_move; output transformer: print_board
B = create_board()
print_board(B)
C = create_coordinate(1, 2)
B = board_fill_position(B, C, 2)
print_board(B)
C = create_coordinate(2, 3)
B = board_fill_position(B, C, 4)
B = board_update_score(B, 48)
print_board(B)
C = create_coordinate(2, 2)
B = board_fill_position(B, C, 8)
print_board(B)
C = create_coordinate(3, 3)
B = board_fill_position(B, C, 2048)
B = board_update_score(B, 2048)
print_board(B)
C = create_coordinate(1, 1)
B = board_fill_position(B, C, 256)
print_board(B)
B = create_board()
C = create_coordinate(1, 2)
B = board_fill_position(B, C, 2)
C = create_coordinate(1, 4)
B = board_fill_position(B, C, 2)
print_board(B)
M = request_move()
S
B = board_reduce(B, M)
C = create_coordinate(1, 4)
B = board_fill_position(B, C, 2)
print_board(B)