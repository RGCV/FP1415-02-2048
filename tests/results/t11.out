[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 0
[ 0 ] [ 2 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 0
[ 0 ] [ 2 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 4 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 48
[ 0 ] [ 2 ] [ 0 ] [ 0 ] 
[ 0 ] [ 8 ] [ 4 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 48
[ 0 ] [ 2 ] [ 0 ] [ 0 ] 
[ 0 ] [ 8 ] [ 4 ] [ 0 ] 
[ 0 ] [ 0 ] [ 2048 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 2096
[ 256 ] [ 2 ] [ 0 ] [ 0 ] 
[ 0 ] [ 8 ] [ 4 ] [ 0 ] 
[ 0 ] [ 0 ] [ 2048 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 2096
[ 0 ] [ 2 ] [ 0 ] [ 2 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
Pontuacao: 0
Introduza uma jogada (N, S, E, W): S
[ 0 ] [ 0 ] [ 0 ] [ 2 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 0 ] [ 0 ] [ 0 ] 
[ 0 ] [ 2 ] [ 0 ] [ 2 ] 
Pontuacao: 0
